packages/core/src/Stores/contract-store.js

```diff
@@ -196,32 +196,30 @@ export default class ContractStore extends BaseStore {
             ) {
                 return;
             }
+            if (!this.barriers_array.length) {
```

Comment:
This helps fix an issue with incorrect initial rendering of barrier range on the first C.Details page load that was exposed after applying chart re-scaling feature.